parsers/www.bonappetit.com.js

```diff
@@ -0,0 +1,5 @@
+exports.title = $ => $('h1').text()
```

Comment:
This is a great use case for a JSON parser, but we would need to update the `title` method to work with a selector. Don't need to do it now.